cerberus/client.py

```diff
@@ -55,6 +56,104 @@ def get_token(self):
         """Returns a client token from Cerberus"""
         return self.token
 
+    def get_roles(self):
+        """Returns all the roles (IAM or User Groups) that can be granted to a safe deposit box.
+
+        Roles are permission levels that are granted to IAM or User Groups.  Associating the id for the write role
+          would allow that IAM or User Group to write in the safety deposit box."""
+        sdb_resp = requests.get(self.cerberus_url + '/v1/role',
+                                headers=self.HEADERS)
+
+        self.throw_if_bad_response(sdb_resp)
+        return sdb_resp.json()
+
+    def get_role(self, key):
+        """Returns id of named role."""
+
+        sdb_resp = requests.get(self.cerberus_url + '/v1/role',
+                                headers=self.HEADERS)
+        self.throw_if_bad_response(sdb_resp)
+        json_resp = sdb_resp.json()
+        for item in json_resp:
+            if key in item["name"]:
+                return item["id"]
+        raise CerberusClientException("Key '%s' not found" % key)
+
+    def list_roles(self):
```

Comment:
I see the comment here, but still not sure why this is needed?

Reply:
It was more as a convenience function.  It returns a more concise result  as a dictionary, than `get_roles()` which returns a list containing dictionaries with more info.

Reply:
That's fine. It's a small method that doesn't seem to hurt anything.